benchmarks/src/main/java/org/apache/druid/benchmark/query/SqlBenchmark.java

```diff
@@ -111,16 +118,27 @@ public void setup()
         .createQueryRunnerFactoryConglomerate();
     final QueryRunnerFactoryConglomerate conglomerate = conglomerateCloserPair.lhs;
     final PlannerConfig plannerConfig = new PlannerConfig();
-
+    final DruidLeaderClient druidLeaderClient = new DruidLeaderClient(
+        EasyMock.createMock(HttpClient.class),
+        EasyMock.createMock(DruidNodeDiscoveryProvider.class),
+        "nodetype",
+        "/simple/leader",
+        new ServerDiscoverySelector(EasyMock.createMock(ServiceProvider.class), "test")
+    )
+    {
```

Comment:
This subclass doesn't mean empty. It just inherits all methods from the parent class. 

Please check my comment below in `PlannerFactory`.